src/Playroom/CodeEditor/CodeEditor.tsx

```diff
@@ -436,6 +267,9 @@ export const CodeEditor = ({ code, onChange, previewCode, hints }: Props) => {
           'Alt-Down': swapLineDown,
           'Shift-Alt-Up': duplicateLine('up'),
           'Shift-Alt-Down': duplicateLine('down'),
+          'Cmd-Alt-Up': addCursorToPrevLine,
+          'Cmd-Alt-Down': addCursorToNextLine,
+          'Cmd-D': selectNextOccurrence,
```

Comment:
It would be good to have some test coverage for these functions (and the ones above). It's a lot of complex code. What's your confidence that these won't break in a minor CodeMirror version bump? We can't rely just on the @askoufis test suite.